test/ripper_parser/sexp_handlers/blocks_test.rb

```diff
@@ -504,11 +504,11 @@
       end
 
       it "works when the fallback value is a keyword" do
-        _("foo rescue next")
+        _("foo rescue return")
```

Comment:
Let's see if Ripper will start supporting this again or if there's a setting. Prism parses this just fine.